addons/hr_timesheet/hr_timesheet_config_settings_views.xml

```diff
@@ -13,6 +13,7 @@
                     <button string="Cancel" type="object" name="cancel" class="oe_link"/>
                 </header>
                 <sheet>
+                    <field name="company_id" invisible="1"/>
```

Comment:
Because you have stuff depending on the company in settings, required on some onchanges. It is simpler to add it in the base view than in child views to avoid the field being added several times.